benchmarks/ps-experiment/README.md

```diff
@@ -0,0 +1,176 @@
+## PS Performance Benchmark
+
+This setup is used to reproduce the PS benchmark. Although the benchmark is carefully crafted to highlight the strengths of the PS use case—with configurations heavily skewed in its favor—we still run it against the AIBrix setup for comparison.
+However, please note that this is a narrow and selective case, not representative of a comprehensive or production-grade workload. It only covers one specific execution path.
+
+> Note: we will publish a comprehensive benchmark scripts and datasets soon.
+
+## Manifest Preparation
+
+We already generated the experiment Yaml, If you like to generate from scratch, follow the steps below.
+
+> Note:
+>
+> 1. The scripts has been verified to be working on VKE, If you need to run on other clouds, you still need to make some changes.
+> 2. Due to testing environments difference, default yaml use `aibrix-container-registry-cn-beijing.cr.volces.com`,
+you can remove this prefix to directly fetch images from dockerhub.
+
+
+| Experiment Name           | Gateway     | Routing Algorithm   | vLLM Version            | KV Cache                    | Explanation                                                                             |
+|---------------------------|-------------|---------------------|-------------------------|-----------------------------|-----------------------------------------------------------------------------------------|
+| aibrix_kvcache_0.6.1.yaml | AIBrix      | Kubernetes          | 0.6.1-aibrix-optimized  | AIBrix client + distributed | AIBrix deployment with KV cache enabled; used to test distributed KV cache behavior.    |
+| aibrix_naive.yaml         | AIBrix      | Kubernetes          | 0.7.0-upstream          | w/o                         | AIBrix baseline setup without optimizations; used as a naive comparison point.          |
+| k8s_stack.yaml            | K8S Service | Kubernetes          | 0.7.0-upstream          | w/o                         | General Kubernetes stack configuration for running inference workloads. K8s Service     |
+| ps_stack.yaml             | PS Router   | Session cache aware | 0.7.0-upstream          | w/o                         | Helm-rendered YAML for the PS (Parameter Server) setup; includes Router, not real naive |
+| ps_k8s_stack.yaml         | PS Router   | Session cache aware | 0.7.0-lmcache-optimized | LM Cache Client lib         | PS full Setup with session aware router and LMCache.                                    |
```

Comment:
for line 24 and 25, experiment name is flipped.

Reply:
nice catch, I will make the change